src/DetailView/DetailView.js

```diff
@@ -1,53 +1,73 @@
+        const dateYear = releaseDateObject.getFullYear()
 
         this.setState({
           averageRating: data.average_rating,
           backdropPath: data.backdrop_path,
-          budget: data.budget,
-          genres: data.genres,
+          budget: data.budget.toLocaleString(),
+          genres: data.genres.join(' '),
           id: data.id,
           overview: data.overview,
           posterPath: data.poster_path,
-          releaseDate: data.release_date,
-          revenue: data.revenue,
+          releaseDate: releaseDateObject,
+          dateData: dateData,
+          dateYear: dateYear,
+          revenue: data.revenue.toLocaleString(),
           runtime: data.runtime,
-          tagline: data.tagline,
           title: data.title
         })
       })
+
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.movieId}/videos`)
+      .then(response => response.json())
+      .then(videoObject => {
+        const data = videoObject.videos[0]
+        const isEmbedAvailable = data === [] ? false : true
+        if (isEmbedAvailable) {
+          this.setState({
+            videoUrl: data.key
+          })
+        }
+      })
   }
 
   render() {
     return <>
-      <p>{this.movieId}</p>
-      <p>{this.state.averageRating}</p>
-      <img className="backdrop" src={this.state.backdropPath} alt={`Artwork for the movie ${this.state.title}`}/>
-      <p>{this.state.budget}</p>
-      <p>{this.state.genres}</p>
-      <p>{this.state.overview}</p>
-      <img className="poster" src={this.state.posterPath} alt={`Artwork for the movie ${this.state.title}`}/>
-      <p>{this.state.releaseDate}</p>
-      <p>{this.state.revenue}</p>
-      <p>{this.state.runtime}</p>
-      <p>{this.state.tagline}</p>
-      <p>{this.state.title}</p>
-      <button onClick={() => {this.closeMovie()}}>Close Movie</button>
+      <section className="details-container" style={{backgroundImage: `linear-gradient(0deg, rgba(0,0,0,1) 0%, rgba(0,0,0,0.75) 50%, rgba(0,0,0,1) 100%),
+url(${this.state.backdropPath})`}}>
+        <button className="back-btn" onClick={() => {this.closeMovie()}}><img className="btn-icon" src={backButton} alt="An arrow pointing left"/></button>
+        <img className="poster" src={this.state.posterPath} alt={`Artwork for the movie ${this.state.title}`}/>
+        <div className="text-container">
+          <h2 className="title">{`${this.state.title} (${this.state.dateYear})`}</h2>
+          <h3 className="genres">{this.state.genres}</h3>
+          <p className="overview">{this.state.overview}</p>
+          <p className="movie-info">{this.state.runtime} minutes</p>
+          <p className="movie-info">release date: {this.state.dateData}</p>
+          <p className="movie-info">budget: ${this.state.budget}</p>
+          <p className="movie-info">box office: ${this.state.revenue}</p>
```

Comment:
I'm thinking this content could also be inside a `<table>` element if we want thing slightly more screen-reader friendly. Also works as-is, but just a thought.

Reply:
I agree, good idea.

Reply:
Updated it to a table in the most recent PR.